src/msw/menu.cpp

```diff
@@ -916,30 +922,47 @@ void wxMenuBar::Refresh()
 
 WXHMENU wxMenuBar::Create()
 {
-    if ( m_hMenu != 0 )
+    if (m_hMenu != 0)
```

Comment:
There is a bunch of whitespace changes going against the code style (perhaps done automatically by a tool). These should be reverted (and the new code formatting should match the expected style).